src/webgpu/shader/validation/parse/static_assert.spec.ts

```diff
@@ -0,0 +1,26 @@
+export const description = `Parser validation tests for static_assert`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { ShaderValidationTest } from '../shader_validation_test.js';
+
+export const g = makeTestGroup(ShaderValidationTest);
+
+const kCases = {
+  no_parentheses: { code: `static_assert true;`, pass: true },
+  left_parenthesis_only: { code: `static_assert(true;`, pass: false },
+  right_parenthesis_only: { code: `static_assert true);`, pass: false },
+  both_parentheses: { code: `static_assert(true);`, pass: true },
+  condition_on_newline: {
+    code: `static_assert
```

Comment:
Can multiline versions have parentheses around the condition?
```
static_assert
   (true);
```

If so can the parantheses be on different lines?
```
static_assert(
   true);
```
or
```
static_assert(
   true
);
```

Possibly a test with a complex condition, just to make sure someone doesn't just supporting parsing true/false?

Reply:
Though I guess with the complex statement that might be covered by the validation tests.

Reply:
Added some cases for multiline with / without parentheses. This is getting into the realm of testing whitespace, which seems like an unbounded set of tests, so I've only done the minimal I feel is necessary here.